Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    BoxCollider2D box;
+    
+    GameObject[] EnemyBandits;
+
+    int currentScene;
+
+    private void Start()
+    {
+        currentScene = SceneManager.GetActiveScene().buildIndex;
+        EnemyBandits = GameObject.FindGameObjectsWithTag("EnemyBandit");        
+    }
+
+    private void Awake()
+    {
+        box = GetComponent<BoxCollider2D>();
+    }
+
+    void OnTriggerStay2D(Collider2D other)
+    {
+        if ((Input.GetKey(KeyCode.UpArrow) || Input.GetKey(KeyCode.W)) && other.tag == "Player")
+        {
+            foreach (GameObject Enemy in EnemyBandits)
+            {
+                Enemy.GetComponent<Rigidbody2D>().constraints = RigidbodyConstraints2D.FreezePositionX;
```

Comment:
Why do we do this? Extract the `foreach` loop in it's own named method and add a summary tag that explains what it's for. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/summary

Reply:
hotkey in visual studio for summaries: type /// into the text editor